src/pages/Home/Home.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import HeroImage from 'assets/hero.svg';
+import SocialButton from 'components/atoms/SocialButton';
+import listItemIcon from 'assets/listItemIcon.svg';
```

Comment:
Hmm Maybe we can create index.tsx for all assets and export them by name?

Reply:
index.ts

```
import HomePageImg_1 from './homepage-image-1.svg';
import listItemIcon from './listItemIcon.svg';

export { HomePageImg_1, listItemIcon };
```

Reply:
assets/images/index.ts
```
export { default as HomePageImg_1 }  from './homepage-image-1.svg';
export { default as ListItemIcon } from './listItemIcon.svg';
```

Could you check it?

Reply:
[https://github.com/mui-org/material-ui/blob/master/packages/material-ui-icons/src/index.js](https://github.com/mui-org/material-ui/blob/master/packages/material-ui-icons/src/index.js)

Reply:
ok

Reply:
Ok. This solution is working.